### PR TITLE
add training example for gke parallelstore blueprint

### PR DESCRIPTION
**Pull request by chengcongdu**
referenced user guide: https://cloud.google.com/parallelstore/docs/tensorflow-sample

sample output: https://paste.googleplex.com/5444375416143872

cloud build: https://pantheon.corp.google.com/cloud-build/builds/1a7bc989-ed73-4103-9359-b91a92f6507e?project=hpc-toolkit-dev&e=GkeGettingStartedCloudShellSolutionsLaunch::GkeGettingStartedCloudShellSolutionsEnabled,GkeOnboardingOverviewTabLaunch::GkeOnboardingOverviewTabEnabled&mods=monitoring_api_prod

### CODE REVIEW

File: examples/gke-storage-parallelstore.yaml [hunk 1]
```diff
@@ -76,32 +76,46 @@ deployment_groups:
...
-
-        echo "Writing seed data to Parallelstore volumes"
-        dd if=/dev/urandom of=/data/parallelstore-pvc-0/${JOB}/${JOB_COMPLETION_INDEX}.dat bs=1K count=1000
-        dd if=/dev/urandom of=/data/parallelstore-pvc-1/${JOB}/${JOB_COMPLETION_INDEX}.dat bs=1K count=1000
-
-        # echo "Hash file and write between the 2 hyerpdisk balanced volumes"
-        # md5sum /data/parallelstore-pvc-0/${JOB}/${JOB_COMPLETION_INDEX}.dat > /data/parallelstore-pvc-1/${JOB}/${JOB_COMPLETION_INDEX}.md5
-        # md5sum /data/parallelstore-pvc-1/${JOB}/${JOB_COMPLETION_INDEX}.dat > /data/parallelstore-pvc-0/${JOB}/${JOB_COMPLETION_INDEX}.md5
-      node_count: 5
+        pip install transformers datasets
```
Comment (ankitkinra) [inline examples/gke-storage-parallelstore.yaml:103]: Is there a way to install all this in python virtual env so it is all cleaned up after customer has completed this job ?

File: examples/gke-storage-parallelstore.yaml [hunk 2]
```diff
@@ -76,32 +76,46 @@ deployment_groups:
...
       name: sample-pool
       zones: [$(vars.zone)]
-      machine_type: n2-standard-4
+      machine_type: n2-standard-16
 
-  ### Parallelstore enabled Job ###
+  # Train a TensorFlow model with Keras and Parallelstore on GKE
+  # Tutorial: https://cloud.google.com/parallelstore/docs/tensorflow-sample
 
   - id: parallelstore-job
```
Comment (ankitkinra) [inline examples/gke-storage-parallelstore.yaml:84]: nit: can you please add which model/training this job will run  ? Or add a comment

File: examples/gke-storage-parallelstore.yaml [hunk 3]
```diff
@@ -76,32 +76,46 @@ deployment_groups:
...
     source: modules/compute/gke-job-template
     use:
     - gke_cluster
     - parallelstore-setup
     settings:
-      image: busybox
+      name: tensorflow
+      image: jupyter/tensorflow-notebook@sha256:173f124f638efe870bb2b535e01a76a80a95217e66ed00751058c51c09d6d85d
+      security_context:
+      - key: runAsUser
```
Comment (ankitkinra) [inline examples/gke-storage-parallelstore.yaml:93]: nit: can you please in a comment, why this security context is necessary. ?

File: examples/gke-storage-parallelstore.yaml (hunk 2 above)
Comment (chengcongdu) [reply examples/gke-storage-parallelstore.yaml:84]: it is already mentioned in line 81, one line above

File: examples/gke-storage-parallelstore.yaml (hunk 3 above)
Comment (chengcongdu) [reply examples/gke-storage-parallelstore.yaml:93]: done

File: examples/gke-storage-parallelstore.yaml (hunk 1 above)
Comment (ankitkinra) [reply examples/gke-storage-parallelstore.yaml:103]: ack

File: examples/gke-storage-parallelstore.yaml (hunk 2 above)
Comment (ankitkinra) [reply examples/gke-storage-parallelstore.yaml:84]: Ack , oops!

**Review by ankitkinra — APPROVED** [review]

**Pull request merged**